JASP-Engine/JASP/R/LScommon.R

```diff
@@ -922,7 +922,7 @@ hdi.density    <- function(object, credMass=0.95, allowSplit=FALSE, ...) {
     }
   }
   if (!is.null(xBlacked))
-    fillColor <- c(fillColor, "black")
+    fillColor <- c(fillColor, "blue")
```

Comment:
If this change was done based on E.J.'s suggestion to use the same colour as in the Distributions module (I saw this somewhere but cannot find the issue now), then this should be `"steelblue"`. Ignore this comment if it's not relevant.

Reply:
yes, you are right. I search the Distribution module code and I found that you use "blue" color somewhere. It looked similar enough to me -I should probably get my color-sight accuracy checked :D